met-web/src/components/metadataManagement/TaxonEditForm.tsx

```diff
@@ -0,0 +1,325 @@
+                            <PresetValuesEditor control={methods.control} name="preset_values" />
+                        </Collapse>
+                    </Grid>
+                    <Grid item>
+                        <Collapse in={taxonType.supportsPresetValues && allowLimiting} timeout="auto" unmountOnExit>
+                            <FormControlLabel
+                                control={
+                                    <Controller
+                                        name="freeform"
+                                        control={control}
+                                        render={({ field: { onChange, value } }) => {
+                                            return (
+                                                <Switch
+                                                    checked={!value && allowLimiting}
+                                                    onChange={(e) => onChange(!e.target.checked)}
+                                                />
+                                            );
+                                        }}
+                                    />
+                                }
+                                label={
+                                    <Box display="flex" alignItems="center">
+                                        {isFreeform ? <ShieldMoon /> : <VerifiedUser />}
+                                        <Box ml={1}>Limit to preset values</Box>
+                                    </Box>
+                                }
+                            />
+                        </Collapse>
+                    </Grid>
+                    <Grid item>
+                        <Collapse in={taxonType.supportsMulti} timeout="auto" unmountOnExit>
+                            <FormControlLabel
+                                control={
+                                    <Controller
+                                        name="one_per_engagement"
+                                        control={control}
+                                        render={({ field: { onChange, value } }) => (
+                                            <Switch checked={!value} onChange={(e) => onChange(!e.target.checked)} />
+                                        )}
+                                    />
+                                }
+                                label={
+                                    <Box display="flex" alignItems="center">
+                                        {isMulti ? <Queue /> : <AddBox />}
+                                        <Box ml={1}>Allow multiple values</Box>
+                                    </Box>
+                                }
+                            />
+                        </Collapse>
+                    </Grid>
+                    <Grid item container alignItems="center" justifyContent="flex-start">
+                        <Button variant="outlined" onClick={() => setSelectedTaxonId(-1)}>
```

Comment:
On the subject of accessibility, if you were a non-sighted person using a keyboard, would it be clear what "cancel" on its own means? Is there enough context on this button for someone not relying on sight to understand what it does? If not, we should add an aria-label with more context.

I'm going to speak to Steve about what's expected of the backend of the app in terms of web accessibility. No need to rush to update/change this or anything